### PR TITLE
Remove WinCE support.

### PR DESCRIPTION
**Pull request by richsalz**
Keeping UWP (Universal Windows Platform), per Nick Banks.

### CODE REVIEW

File: crypto/bio/bss_log.c [hunk 1]
```diff
@@ -22,8 +22,7 @@
 #include "bio_local.h"
 #include "internal/cryptlib.h"
 
-#if defined(OPENSSL_SYS_WINCE)
-#elif defined(OPENSSL_SYS_WIN32)
+#if   defined(OPENSSL_SYS_WIN32)
 #elif defined(__wasi__)
```
Comment (botovq) [inline crypto/bio/bss_log.c:26]: This is weird. Should this be `#if defined(__wasi__)`?

File: crypto/ui/ui_openssl.c [hunk 2]
```diff
@@ -61,9 +61,7 @@
 
 # ifdef WIN_CONSOLE_BUG
 #  include <windows.h>
-#  ifndef OPENSSL_SYS_WINCE
 #   include <wincon.h>
```
Comment (botovq) [inline crypto/ui/ui_openssl.c:64]:
```suggestion
#  include <wincon.h>
```

File: include/internal/e_os.h [hunk 3]
```diff
@@ -165,11 +161,7 @@ FILE *__iob_func(void);
 #  ifndef R_OK
 #   define R_OK        4
 #  endif
-#  ifdef OPENSSL_SYS_WINCE
-#   define DEFAULT_HOME  ""
-#  else
 #   define DEFAULT_HOME  "C:"
```
Comment (botovq) [inline include/internal/e_os.h:164]:
```suggestion
#  define DEFAULT_HOME  "C:"
```

**Review by botovq — COMMENTED** [review]
Only a few very tiny things.

File: include/internal/sockets.h [hunk 4]
```diff
@@ -208,10 +208,6 @@ typedef size_t socklen_t;        /* Currently appears to be missing on VMS */
 # endif
 
 /* also in apps/include/apps.h */
-# if defined(OPENSSL_SYS_WIN32) || defined(OPENSSL_SYS_WINCE)
-#  define openssl_fdset(a, b) FD_SET((unsigned int)(a), b)
```
Comment (botovq) [inline include/internal/sockets.h:212]: Sorry, I missed this previously. I think you need to keep this for `OPENSSL_SYS_WIN32` and probably add parentheses for the sake of hygiene:

```C
# if defined(OPENSSL_SYS_WIN32)
#  define openssl_fdset(a, b) FD_SET((unsigned int)(a), (b))
# else
#  define openssl_fdset(a, b) FD_SET((a), (b))
# endif
```

**Review by botovq — COMMENTED** [review]
Found one more thing, other than that this looks good.

File: include/internal/sockets.h (hunk 4 above)
Comment (richsalz) [reply include/internal/sockets.h:212]: fixup pushed.  you have nothing to apologize for, I should have done a better job :)

**Review by botovq — APPROVED** [review]

**Comment by richsalz** [comment]
yamc (yet another changes.md merge conflict, please re-approve)

**Review by botovq — APPROVED** [review]
Heh. No surprise there. yaa (yet another approval)

**Pull request merged**